sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -254,6 +254,12 @@ def from_infra_object_proto(infra_object_proto: InfraObjectProto) -> Any:
             region=infra_object_proto.dynamodb_table.region,
         )
 
+    @staticmethod
+    def from_proto(dynamodb_table_proto: DynamoDBTableProto) -> Any:
+        return DynamoDBTable(
+            name=dynamodb_table_proto.name, region=dynamodb_table_proto.region,
+        )
+
```

Comment:
not sure if this is relevant, but I _think_ that dynamo would error if you try to update a feature view: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_CreateTable.html

Reply:
yup am tracking this for future feast plan work